packages/errors/src/errors.ts

```diff
@@ -1551,6 +1551,14 @@ export const AllCypressErrors = {
       the devServer config as a ${fmt.highlight(devServerConfigFile)} option.
     `
   },
+
+  TESTING_TYPE_NOT_CONFIGURED: (testingType: TestingType) => {
+    return errTemplate`
+      The testing type selected (${fmt.highlight(testingType)}) is not configured in your config file.
+
+      Please update it, and try running it again.
```

Comment:
Do we also want to direct users to the configuration docs here? https://on.cypress.io/configuration

We do so for other errors, like INVALID_CONFIG_OPTION.

Reply:
Yes, this would be good to add also.

Reply:
[90070e3](https://github.com/cypress-io/cypress/pull/22000/commits/90070e3ac647ab651383eb9f6930521d9e7c0fe8)